testsuite/features/secondary/https_connection.feature

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2015-2025 SUSE LLC
```

Comment:
That's a new one, so you don't need to write 2015-2025, we normally just keep the year of creation and year of last modification.